lib/polyfill/pip_webkit.js

```diff
@@ -47,6 +47,16 @@ shaka.polyfill.PiPWebkit = class {
       return;
     }
 
+    const video =
+      /** @type {!HTMLVideoElement} */ (document.createElement('video'));
```

Comment:
Maybe, you can use:
`const video = shaka.util.Platform.anyMediaElement_();`

Reply:
Thanks @avelad, didn’t know about this one, but in any case it is private so doesn’t seem like it was intended for external use.